module/VuFind/src/VuFind/Recommend/MapSelection.php

```diff
@@ -198,6 +208,19 @@ public function init($params, $request)
     {
     }
 
+    /**
+     * Get the basemap configuration settings.
+     *
+     * @return array
+     */
+    public function getBasemap()
+    {
+        $basemapParams = [];
```

Comment:
This might be a little more concise as:

<pre>
return [
    $this->basemapOptions['basemap_url'],
    $this->basemapOptions['basemap_attribution']
];
</pre>

Reply:
Thanks. I'll make the adjustment.

Reply:
Thanks! I think there's one other place with identical code that can also be adjusted.